android/build.gradle

```diff
@@ -34,11 +34,11 @@ dependencies {
 
   implementation fileTree(include: ['*.jar'], dir: 'libs')
   implementation 'com.facebook.react:react-native:+'
-  implementation 'com.android.support:support-v4:27.1.0'
-  implementation 'com.android.support:appcompat-v7:27.1.0'
+  implementation 'com.android.support:support-v4:28.0.0'
+  implementation 'com.android.support:appcompat-v7:28.0.0'
   implementation "com.google.android.gms:play-services-wallet:$googlePlayServicesVersion"
   implementation "com.google.firebase:firebase-core:$firebaseVersion"
-  implementation 'com.stripe:stripe-android:8.1.0'
+  implementation "com.github.khealth:stripe-android:google-pay-SNAPSHOT"
```

Comment:
Is there no alternative to this fork?

Reply:
at this point yes. see my point 1.

Reply:
it's very unfortunate. i promise that as soon as a release comes out that works i'll update